### PR TITLE
reuse github action

### PR DESCRIPTION
**Pull request by kon-angelo**
**What this PR does / why we need it**:
Add automated test for license compliance

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```improvement user
NONE
```

### CODE REVIEW

**Review by AxiomSamarth — APPROVED** [review]

File: .reuse/dep5 [hunk 1]
```diff
@@ -89,6 +90,10 @@ Files: vendor/github.com/go-openapi/swag/*
 Copyright: 2015 go-swagger
 License: Apache-2.0
 
+Files: vendor/github.com/go-task/slim-sprig/*
```
Comment (msohn) [inline .reuse/dep5:93]: is this addition related to this PR ?

Comment (kon-angelo) [reply .reuse/dep5:93]: I guess this was a dependency that was never documented properly. 

When I run the tests in my fork the reuse tests were running (and here are not but I am not aware if it because I merging from my fork to the upstream) and the reuse test was failing because of the missing license for `slim-sprig`. I added this so that the test will be in a green state when the gh-action is merged.

**Review by dkistner — APPROVED** [review]
/lgtm
